src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -635,6 +652,13 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_CONSOLE:
                         arg_console = true;
+                        if (optarg) {
+
+                                if (isempty(optarg))
+                                        return -EINVAL;
```

Comment:
Should really log a friendly explanation to the user here, that the passed argument is invalid. Just dying silently is not OK.